modules/utils/package/README.md

```diff
@@ -1,38 +1,39 @@
-|numpy| 1.22.3     |    python_packages|
-|Pillow| 9.1.1      |    python_packages|
-|python-dateutil|      2.8.2 | python_packages        |
-|pytz| 2022.1     |    python_packages|
-|requests| 2.26.0     |    python_packages|
-|six| 1.16.0     |    python_packages|
-|typing_extensions| 4.2.0      |    python_packages|
-|urllib3| 1.26.7     |    python_packages|
-|pbkdf| 2-1.3      |    python_packages|
-|wifi| 0.3.8      |    python_packages|
-|cryptography| 39.0.0     |    python_packages|
-|scitools_pyke| 1.1.1      |    python_packages|
-|torchaudio| 0.11.0     |    machine_learning2|
-|torchvision| 0.12.0     |    machine_learning2|
-|scikit-learn| 1.1.1      |    machine_learning2|
-|scipy| 1.8.1      |    machine_learning2|
-|xgboost| 1.6.1      |    machine_learning2|
-|torch| 1.11.0     |    machine_learning_packages|
-|dpkt| 1.9.7.2    |    machine_learning_packages|
-|cycler| 0.11.0     |    machine_learning_packages|
-|fonttools| 4.33.3     |    machine_learning_packages|
-|joblib| 1.1.0      | machine_learning_packages |
-|kiwisolver| 1.4.3      | machine_learning_packages |
-|matplotlib| 3.5.2      | machine_learning_packages |
-|packaging| 21.3       | machine_learning_packages |
-|pyparsing| 3.0.9      | machine_learning_packages |
-|sklearn| 0          | machine_learning_packages |
-|threadpoolctl| 3.1.0      | machine_learning_packages |
-|pypcap|            | machine_learning_packages |
+|Package         | Version   |    Location|
+|:----|:----------|:----|
+|pandas| 1.4.2     | python_packages |
+|certifi| 2022.12.7 | python_packages     |
+|charset-normalizer| 2.0.7     |    python_packages|
+|idna| 3.2       |python_packages|
+|numpy| 1.22.3    |    python_packages|
+|Pillow| 9.3.0     |    python_packages|
+|python-dateutil| 2.8.2     | python_packages        |
+|pytz| 2022.1    |    python_packages|
+|requests| 2.26.0    |    python_packages|
+|six| 1.16.0    |    python_packages|
+|typing_extensions| 4.2.0     |    python_packages|
+|urllib3| 1.26.7    |    python_packages|
+|pbkdf| 2-1.3     |    python_packages|
+|wifi| 0.3.8     |    python_packages|
+|cryptography| 39.0.1    |    python_packages|
+|scitools_pyke| 1.1.1     |    python_packages|
+|torchaudio| 0.11.0    |    machine_learning2|
+|torchvision| 0.12.0    |    machine_learning2|
+|scikit-learn| 1.1.1     |    machine_learning2|
+|scipy| 1.10.1    |    machine_learning2|
+|xgboost| 1.7.0     |    machine_learning2|
+|torch| 1.13.1    |    machine_learning_packages|
+|dpkt| 1.9.7.2   |    machine_learning_packages|
+|cycler| 0.11.0    |    machine_learning_packages|
+|fonttools| 4.33.3    |    machine_learning_packages|
+|joblib| 1.2.0     | machine_learning_packages |
+|kiwisolver| 1.4.3     | machine_learning_packages |
+|matplotlib| 3.5.2     | machine_learning_packages |
+|packaging| 21.3      | machine_learning_packages |
+|pyparsing| 3.0.9     | machine_learning_packages |
+|sklearn| 0         | machine_learning_packages |
```

Comment:
sklearn is deprecated package and scikit-learn should be used instead

Reply:
@Willtl please advise, and take the necessary actions